controlplane/src/core/repositories/OrganizationRepository.ts

```diff
@@ -948,36 +952,38 @@ export class OrganizationRepository {
       );
   }
 
-  public async addOrganizationBilling(input: { organizationID: string; email: string; plan: string }) {
-    await this.db
-      .insert(organizationBilling)
-      .values({
-        organizationId: input.organizationID,
-        plan: input.plan,
-        email: input.email,
-      })
-      .execute();
-  }
-
-  public async getOrganizationLimits(input: { organizationID: string }) {
+  public async getOrganizationFeatures(input: {
+    organizationID: string;
+  }): Promise<{ [key in FeatureIds]: number | boolean }> {
     const features = await this.getFeatures({ organizationId: input.organizationID });
 
-    // fallback limits @todo: think of a better way to make this dynamic
-    const limits: Record<string, number> = {
-      'analytics-retention': 7,
-      'tracing-retention': 7,
-      'changelog-retention': 7,
-      'breaking-change-retention': 7,
-      'trace-sampling-rate': 0.1,
-      requests: 10,
+    // Full list of features with default values
+    const list: { [key in FeatureIds]: number | boolean } = {
+      'analytics-retention': 30,
+      'tracing-retention': 30,
+      'changelog-retention': 30,
+      'breaking-change-retention': 90,
+      'trace-sampling-rate': 1,
+      'federated-graphs': 30,
+      users: 25,
+      requests: 30,
+      rbac: false,
```

Comment:
Please double check this doesn't break feature checks in the organization settings (rbac and oidc)

Reply:
It doesn't for rbac we have a desperate call and for oidc nothing has changed only types has been improved.